flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HivePartitionFetcherContextBase.java

```diff
@@ -137,21 +133,17 @@ public List<ComparablePartitionValue> getComparablePartitionValueList() throws E
                 }
                 break;
             case CREATE_TIME:
+                Map<List<String>, Long> partValuesToCreateTime = new HashMap<>();
                 partitionNames =
                         metaStoreClient.listPartitionNames(
                                 tablePath.getDatabaseName(),
                                 tablePath.getObjectName(),
                                 Short.MAX_VALUE);
-                List<String> newNames =
-                        partitionNames.stream()
-                                .filter(
-                                        n ->
-                                                !partValuesToCreateTime.containsKey(
-                                                        extractPartitionValues(n)))
-                                .collect(Collectors.toList());
                 List<Partition> newPartitions =
```

Comment:
Since we're not trying to get only "new" partitions, please rename this variable to reflect that.

Reply:
OK, thanks. I have modified the variable names to be more realistic.